pfio/v2/fs.py

```diff
@@ -338,25 +334,24 @@ def from_url(url: str, **kwargs) -> 'FS':
 
         create (bool): Create the specified path doesn't exist.
 
-        reset_on_fork (bool): Reset stateful objects (e.g. connection
-            to the remote system) after fork.
-
     .. note:: Some FS resouces won't be closed when using this
         functionality.
 
     .. note:: Pickling the FS object may or may not work correctly
         depending on the implementation.
 
     '''
-    parsed = urlparse(url)
-    force_type = kwargs.pop('force_type', None)
+    if kwargs.pop('reset_on_fork', None) is not None:
+        print("reset_on_fork is deprecated. PFIO resets on fork by default")
```

Comment:
Please use `warnnings.warn(...)` to emphasize it's warning and let the application control it. Maybe [DeprecationWarning](https://docs.python.org/3/library/warnings.html#warnings.warn) suits this case.

Reply:
Fixed by https://github.com/pfnet/pfio/pull/297/commits/296804114f1ee9b4dad4210e3b2b5476e64a4a0a